wiki/osu!supporter/zh.md

```diff
@@ -1,66 +1,155 @@
-| 12 | 26 (最多-46%) |
-| 24 | 52 (最多-46%) |
+### 用户名颜色
 
-- 费用由个人账户支付。没有团体馈赠的促销活动。
+::: Infobox
+![](img/chat.png "一位 osu! 支持者正在频道中聊天")
+:::
 
-### 确认
+未过期的支持者们在[聊天窗口](/wiki/Client/Interface/Chat_console)中拥有高亮的黄色用户名。
 
-默认情况下，将显示你的帐户。如果你不想要 osu!supporter，点击“赠与另一个玩家”并输入该玩家的账户名称。如果你想赠送给多个帐户，请点击“结帐”按钮旁边的“添加另一个标志”按钮。确认帐户后，拖动帐户下方的白色按钮，可以确定帐户需要多长时间，以及总共需要支付的费用。你可以为不同的账户支付不同的时间的 supporter 以及不同的费用的 supporter。
+### 可编辑的玩家信息页区块
 
-### 付款
+支持者们会获得一个名为 `个人介绍` 的玩家信息页区块。该区块内容可以由玩家自行编辑，文本格式使用 [BBCode](/wiki/BBCode)。该功能是永久的，不会因为支持者标签的过期而消失。
 
-确认期限和账户后，点击“添加到购物车”，确认无误后点击 “Checkout”。在此之后，你可以选择不同的支付方式。建议使用 [PayPal](https://www.paypal.com) 或信用卡。以及其他支付方式（例如银联支付和微信支付），但他们可能收取你手续费。如果你支付的费用低于“总费用”（包括使用非 PayPal 时的额外费用），则会支付失败。
+`个人介绍` 区块适合各种情况，比如：
 
-### 有效时间
+- 用联动横幅或个人信息图表来给你的资料加点料
+- 为玩家们的生涯提供些许建议
+- 展示最爱的谱面，谱师，或者其他重要的东西
 
-![supporter 用户与非 supporter 用户的对比](img/status.jpg?1 "supporter 用户与非 supporter 用户的对比")
+### 个人资料封面
 
- 交易完成后，你的个人资料的近期活动将会显示“{某玩家}已成为 osu!supporter - 感谢你的慷慨解囊！”（第一次成为 osu!supporter 时）或 “{某玩家} 再次选择捐赠 osu! - 感谢你的慷慨解囊！”（你以前曾是 osu!supporter 时）。如果你为自己购买了 osu!supporter，则可以在你的个人资料图片下方看到 osu!supporter 徽章。想知道 osu!supporter *何时*到期，请转到 [osu!supporter 页面](https://osu.ppy.sh/home/support)。
+支持者们可以上传封面来进一步定制他们的个人资料：
 
-哦，你还会收到 osu! 的“感谢”信！就像上面那样。
+- 按下个人资料封面右下角的铅笔图标。
+- 从默认封面中选择一个，或上传自己的封面（其尺寸应为 2400 x 640）。
+
+支持者标签过期后，封面会被保留，但无法提交更改。
+
+### 一次免费的改名
+
+*注意：关于可向账号管理团队请求的免费细微修改，请参见：[帮助中心/账号 § 我可以修改用户名么？](/wiki/Help_centre/Account#用户名更改)*
+
+在[通常条件](/wiki/Help_centre/Account#用户名更改)下，首次获得支持者标签后可以免费获得一次改名机会。
+
+### osu!direct
+
+osu!direct 是供支持者使用的游戏内谱面搜索和下载引擎。可以在主菜单点击屏幕右侧垂直的 `osu!direct` 标识进入。除了一个内置的谱面列表，osu!direct 还提供其他不需要切出游戏就可以下载新谱面的方式：
+
+- 如果聊天窗口中的谱面链接指向 osu! 网页的谱面，可以直接在游戏内弹出下载提示
+- 可以在[多人游戏](/wiki/Gameplay/Multiplayer)房间或[旁观](/wiki/Gameplay/Spectating)中使用的自动谱面下载器。这是一个游戏内[可选设置](/wiki/Client/Options#integration)
+- 在被旁观时，可自动向 `#spectator` 聊天窗口发送当前谱面链接，这也是一个游戏内[可选设置](/wiki/Client/Options#online)
+
+### 额外排行榜
+
+支持者们可以在游戏或网页中访问若干个特殊的谱面[排行榜](/wiki/Beatmap#leaderboards)：
+
+- 按 [Mod](/wiki/Game_modifier) 组合筛选的全球排行榜
+- 国内/区内排行榜
+- 以及一个可以和朋友们竞争的好友排行榜
+
+### 提高限额
+
+osu! 为支持者们放宽各种在线功能的限额：
+
+| 功能 | 普通玩家限额 | 支持者限额 |
+| :-- | :-: | :-: |
+| [Pending 谱面](/wiki/Beatmap/Category#work-in-progress-and-pending) | `4 + min(Ranked 谱面, 2)` | `8 + min(Ranked 谱面, 4)` |
+| 在线收藏的谱面 | 100 | 1000 |
+| 好友数量 | 250 | 500 |
+
+此外，支持者也会获得更宽松的谱面下载限额。
+
+### 额外皮肤元素
+
+获得支持者标签后，osu! 外观界面的一些视觉和音效元素可以加入到皮肤当中：
+
+| 元素 | 简介 |
+| :-- | :-- |
+| `menu-background.jpg` | 主菜单背景图 |
+| `welcome_text.png` | 游戏启动时出现的 "welcome" 文本 |
+| `welcome.wav` | 游戏启动时播放的 "welcome to osu!" 音效 |
+| `seeya.wav` | 游戏关闭时播放的 "see ya next time" 音效 |
+
+获取更多信息，请参见 [皮肤/外观界面 § 主菜单](/wiki/Skinning/Interface#main-menu) 和 [皮肤/皮肤音效 § 主菜单](/wiki/Skinning/Sounds#main-menu)。
+
+### 额外搜索
+
+::: Infobox
+![](img/beatmap-search.png "在谱面搜索页中拥有两个新的过滤器")
+:::
+
+支持者们可以使用额外的[谱面搜索](https://osu.ppy.sh/beatmapsets)过滤器：
+
+- 玩过（或没玩过）的谱面
+- 按[成绩](/wiki/Gameplay/Grade)筛选谱面
+
+### 在 osu! 测试版中进行多人游戏
+
+支持者们可以在抢先体验的“测试版”更新源中使用多人游戏功能。
```

Comment:
考虑到Disambiguation/Version还没有正式译名，最好看看术语表，天比较晚了这点我就没确认。。。

Reply:
客户端内“cutting edge”被翻为`测试版`

Reply:
use what's displayed in-game. it doesn't need quotation marks (`"`) either if it's better without.

Reply:
@Kotoki1337 居然现在是这样的，那就还用测试版吧……那这段可以keep，或者你看着语序改?

-----------------------------------

@Walavouchey the English version of this part are follows:
> Supporters may access multiplayer from builds on the early access "cutting edge" release stream.

To Keep insist with [Source File](https://github.com/ppy/osu-wiki/blob/master/wiki/osu!supporter/en.md), I suggest to keep the quotation marks (`"`), if it surely needn't , updated English version first , then fixed on this PR may better.
If necessary, I think translation of a article should give readers reading experience same as "a native speaker reading original article". so "glitch" in original article should also presented on translation version.

Reply:
The meaning needs to be the same, but not necessarily the formatting. There is nothing wrong with the quotation marks in English, but if they're not needed in Chinese, then it's ok to omit them. Either way, that's your call to make, because I don't know Chinese.

Reply:
Well, in my opinion, keeping that double quotation is fine. waiting for more reviewer for avoid subjective is also a opinion.

Reply:
I chose to keep it, I thought of some solutions for quotation marks but all of them are not compliance with Chinese criteria.